### DISCUSSION TITLE
Updated Icon Libraries

### DISCUSSION THREAD

**Discussion by half5cat** [carbon-design-system/carbon#8185] [Help] [answered]
Where can I find the most recent version of the icon library for Sketch? The ones referenced here were last updated 15 months ago: https://www.carbondesignsystem.com/designing/kits/sketch

**Comment by laurenmrice** [comment, marked as answer]
That is the most recent version of the Icon Sketch libraries. Yes, they have not been updated in awhile because we've been having some technical difficulties with our Sketch Plugin that generates our icon assets from code into Sketch. However, the problem is almost fixed and we should be releasing an Icon Library update soon with all icons. 

In the meantime you can download individual icons from our website https://www.carbondesignsystem.com/guidelines/icons/library/.

**Reply by half5cat** [reply]
Thanks for the reply and explanation, @laurenmrice 👍

**Reply by laurenmrice** [reply]
Update: The sketch icon libraries have been updated with the full set of icons!

**Reply by half5cat** [reply]
@laurenmrice Thanks for the update and for letting me know!
